spec/models/video_spec.rb

```diff
@@ -1,11 +1,37 @@
 require 'spec_helper'
 
 describe Video do
-  it "saves itself" do
-    video = Video.new(title: "Test title", description: "A fish and ham sandwich", small_cover_url: "/public/tmp/monk.jpg", large_cover_url: "/public/tmp/monk_large.jpg")
-    video.save
-    #Video.first.title.should == "Test title"
-    expect(Video.first).to eq(video)
+  it { should belong_to(:category)}
```

Comment:
I would recommend to be consistent in hash notation with `{ should belong_to(:category) }` both spaces around curly brackets. Also since you are using `expect(...).to` syntax I would recommend sticking to that here as well. `should` syntax is old and new way is `is_expected.to` - so in this case it would `it { is_expected.to belong_to(:category) }`